cabal.project

```diff
@@ -23,7 +23,6 @@ constraints: text -simdutf
 
 if os(mingw32)
   constraints: zlib +bundled-c-zlib,
-               lzma +static,
                text -simdutf,
```

Comment:
`xz -system-xz` isn't necessary here too?

Reply:
No. The `cabal.project` is for development. Linking to system libs if they exist is fine.